src/bl/userFacade.ts

```diff
@@ -98,4 +110,55 @@ export class UserFacade {
             throw new BadRequest(e)
         }
     }
+
+    static async reset(email: string): Promise<User> {
+        if (!email) {
+            throw new BadRequest('This user does not exist')
+        }
+        try {
+            const user = await UserFacade.getByEmail(email)
+            const token = uuid('photon.igpolytech.fr', uuid.DNS)
+            user.resetToken = token
+            user.resetTimeStamp = new Date()
+            console.log(user.resetTimeStamp)
```

Comment:
???

Reply:
It was for the test, the `console.log()`. I will remove it.